app/models/expense.rb

```diff
@@ -14,6 +14,15 @@ class Expense < ApplicationRecord
   belongs_to :account
   belongs_to :category
 
+  scope :by_year, ->(year) { where('extract(year from due_at) = ?', year) }
+  scope :by_month, ->(month) { where('extract(month from due_at) = ?', month) }
+  scope :by_due, lambda { |year, month|
```

Comment:
Pq nos de cima vc usou -> e aqui lambda?
Nem lembro mais, mas será que assim não funciona?
```suggestion
  scope :by_due, -> { |year, month|
```

Reply:
Coisas do rubocop, quando é 1 linha ele pede pra usar `->` e quando for 2 ou mais linhas ele pede pra usar `lambda`